client/src/router/index.js

```diff
@@ -42,6 +43,11 @@ export default new Router({
       path: '/callback',
       name: 'callback',
       component: AuthCallback
+    },
+    {
+      path: '/playlist/:playlist_id',
+      name: 'playlist',
+      component: Playlist
```

Comment:
Add the `props: true` property so we can pass in the `playlist_id` from Home.vue as described in the comment.